docs/conf.py

```diff
@@ -113,7 +113,7 @@
 # The theme to use for HTML and HTML Help pages.  See the documentation for
 # a list of builtin themes.
 #
-html_theme = 'sphinx_rtd_theme'
+html_theme = 'sphinx_symbiflow_theme'
```

Comment:
Does this need to be installed somehow?

Reply:
Yes, it's in the requirements.txt

Reply:
Just like sphinx_rtd_theme was. It's a 1:1 replacement

Reply:
You need to change the requirements.txt file then?

Reply:
Yup, forgot to commit the requirements file, sorry